packages/core/src/utils.ts

```diff
@@ -242,6 +242,9 @@ export function arrayBufferToBase64(arrayBuffer: ArrayBuffer): string {
 }
 
 export function capitalize(word: string): string {
+  if (word.toLowerCase() === 'id') {
```

Comment:
kind of a smell but I'll allow it.

Reply:
Yeah.... I'd be open to a better way to do this